earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -362,6 +338,8 @@ void PacketGenAssetVersionImplD::UpdateChildren(
                   prevLevelVer.Ref() : SharedString() /* cached blend alpha */));
       }
 
+      levInputs.erase(std::remove(levInputs.begin(), levInputs.end(), ""));
+      levInputVers.erase(std::remove(levInputVers.begin(), levInputVers.end(), nullptr));
```

Comment:
I think you need to include `levInputs.end()` and `levInputVers.end()` in the calls to `erase()`. Otherwise it will only erase once item. That will only be correct in cases where there is only one blank item in the vector.